models/GroupUserDetails.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+    const GroupUserDetails = sequelize.define('GroupUserDetails', {
+
+
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            unique: true,
+        },
+        description: {
+            type: DataTypes.STRING,
+            unique: false,
+        },
+        photo_url: DataTypes.STRING,
+        color: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            unique: true
```

Comment:
Let's remove `unique` here since that'll prevent other users in other groups from having the same color.